doc/classes/Image.xml

```diff
@@ -333,6 +333,24 @@
 				Loads an image from the binary contents of a PNG file.
 			</description>
 		</method>
+		<method name="load_svg_from_buffer">
+			<return type="int" enum="Error" />
+			<param index="0" name="buffer" type="PackedByteArray" />
+			<param index="1" name="scale" type="float" default="1.0" />
+			<description>
+				Loads an image from the binary contents of a SVG file.
+				[b]Note:[/b] This method is only available in [b]svg_enabled[/b] builds.
+			</description>
+		</method>
+		<method name="load_svg_from_string">
+			<return type="int" enum="Error" />
+			<param index="0" name="svg_str" type="String" />
+			<param index="1" name="scale" type="float" default="1.0" />
+			<description>
+				Loads an image from the string contents of a SVG file.
+				[b]Note:[/b] This method is only available in [b]svg_enabled[/b] builds.
```

Comment:
```suggestion
				[b]Note:[/b] This method is only available in engine builds with the SVG module enabled. By default, the SVG module is enabled, but it can be disabled at build-time using the [code]module_svg_enabled=no[/code] SCons option.
```

Reply:
Thanks.